app/views/layouts/_clear_sort.html.erb

```diff
@@ -0,0 +1,7 @@
+<% unless local_assigns[:sort].nil? %>
```

Comment:
Not a critique. Just wondering, is it easier to do this verification here or in JS?

Reply:
Definitely easier serverside

Plus it makes the site more functional without JS without adding too much backend complexity

Reply:
Ah ok ok, nice 👌🏿 Imma try and use that in my code if I have any JS stuff to consider.